src/typ/tests.rs

```diff
@@ -488,9 +488,9 @@ fn infer_bitstring_error_test() {
     );
 
     assert_error!(
-        "case <<1>> { <<1:bitstring, _:binary>> -> 1 }",
+        "case <<1>> { <<1:bit_string, _:binary>> -> 1 }",
         Error::BinarySegmentMustHaveSize {
-            location: SrcSpan { start: 15, end: 26 },
+            location: SrcSpan { start: 15, end: 27 },
```

Comment:
I think it's ok to change these because this is the formatting for highlighting errors, that are now one character longer

Reply:
Looks good :)